hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/Main.java

```diff
@@ -0,0 +1,36 @@
+package org.apache.mesos.hdfs.scheduler;
+
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import org.apache.mesos.hdfs.config.ConfigServer;
+
+/**
+ * Main entry point for the Scheduler.
+ */
+public final class Main {
+
+  private SchedulerExceptionHandler schedulerExceptionHandler = new SchedulerExceptionHandler();
+
+  // utility class
```

Comment:
Do we need this private constructor? Also, what is the comment //utility class for